contents/docs/data/events.mdx

```diff
@@ -114,6 +114,64 @@ Most users of PostHog will want to combine their back-end data, such as user inf
 -   JS [snippet](/docs/integrate)
 -   Client- and server-side [libraries](/docs/integrate)
 
+## Events with person processing
+
+By default all events create or update an associated [person profile](/docs/data/persons). This allows for the following features:
+
+- Filtering on person properties
+- Merging identities across distinct_ids (ie merging anonymous users with your logged in user)
+- Cohort creation
+- Feature flag targeting
+- Tracking initial UTM tags and referrers over time
+
+However, there are many usecases that don't require this person processing. Because events that don't do person processing are cheaper for us to ingest, the pricing for these events is lower.
+
+Some examples of usecases that often don't require user profiles
+- Tracking a marketing website
```

Comment:
```suggestion
- Tracking a marketing website where you don't care about conversions to signup
```

Or something like this. I think we'd still like to have person processing on for posthog.com, yes? Otherwise we lose the identity merging, so can't track across that barrier.

Reply:
Maybe "tracking logged-out users on a marketing website", that's the use case for identified_only

Reply:
I think marketing website is clearest here, we explain the tradeoffs further down